pkg/extensions/sync/destination.go

```diff
@@ -29,26 +30,86 @@ import (
 	storageTypes "zotregistry.dev/zot/pkg/storage/types"
 )
 
+// Platform represents an OS/architecture combination
+type Platform struct {
+	OS           string
+	Architecture string
+}
+
+// ParsePlatform parses a platform string into a Platform struct
+// The string can be in the format "os/arch" or just "arch"
+func ParsePlatform(platform string) Platform {
+	parts := strings.Split(platform, "/")
+	if len(parts) == 2 {
```

Comment:
Note there are also cases for 2 `/` separators, meaning 3 parts, if variant is specified in the platform, for example:

```
    "Descriptor": {
      "mediaType": "application/vnd.docker.distribution.manifest.v2+json",
      "digest": "sha256:116d243c6346c44f3d458e650e8cc4e0b66ae0bcd37897e77f06054a5691c570",
      "size": 742,
      "platform": {
        "architecture": "arm",
        "os": "linux",
        "variant": "v7"
      }
    },
```

This should be taken into account.

Reply:
Added in https://github.com/project-zot/zot/pull/3128/commits/dd6a4cf955b6defac1efcb3838d35113c333cd90